Tests/RestKitTests/JSONValueTests.swift

```diff
@@ -120,6 +133,19 @@ class JSONTests: XCTestCase {
         let expected = "{\"key\":0.5}"
         XCTAssertEqual(json.sorted(), expected.sorted())
     }
+    
+    func testEncodeDate() {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        let stringDate = "2018-05-30"
```

Comment:
The dates we encode/decode should all be ISO 8601 date strings, e.g. "2018-05-17T18:45:32.189Z".